tests/tests_integration/test_api/test_files.py

```diff
@@ -21,6 +37,18 @@ def new_file():
     assert COGNITE_CLIENT.files.retrieve(id=res.id) is None
 
 
+@pytest.fixture(scope="class")
+def new_file_with_geoLocation(mock_geo_location):
+    res = COGNITE_CLIENT.files.upload_bytes(content="blabla", name="geoLocationFile", geo_location=mock_geo_location)
+    while True:
+        if COGNITE_CLIENT.files.retrieve(id=res.id).uploaded:
+            break
+        time.sleep(0.5)
```

Comment:
What is this for?

Reply:
Copied it from the new_file_with_label() fixture below, but I believe it's because the file is not always available immediately after upload.